.changeset/thin-steaks-trade.md

```diff
@@ -0,0 +1,45 @@
+---
+'@backstage/plugin-catalog': patch
+---
+
+Update the default entity page extension component to support grouping multiple entity content items in the same tab.
+
+Disable a default group via configuration:
+
+```diff
+# app-config.yaml
+app:
+  extensions:
+    # Pages
++   - page:catalog/entity:
++       config:
++         groups:
++           deployment: false
+```
+
+Change a default group title via configuration:
+
+```diff
+# app-config.yaml
+app:
+  extensions:
+    # Pages
++   - page:catalog/entity:
++       config:
++         groups:
++           deployment: Infrastructure # this is overriding the default group title
```

Comment:
Kinda feeling it's worth having a `title: ` key in there for readability tbh, so you'd need to have

```yaml
groups:
  deployment:
    title: Infrastructure
```

Happy to discuss though

Reply:
Done.